app/__init__.py

```diff
@@ -30,7 +31,7 @@
 from .data.routes import data_router
 from .rvo.routes import router as rvo_router
 
-cors_config = CORSConfig(allow_origins=["*"])
+cors_config = CORSConfig(allow_origins=[env.PUBLIC_APP_URL], allow_credentials=True)
```

Comment:
This is required to set cookie with credentials from backend

Reply:
I wonder if we need something specific for the mobile apps? They're going to provide the token through the header, right? As a `Bearer` token maybe?
If that's the case, I believe we also need to allow that explicitely?